### ISSUE TITLE
in-app chat with tiledesk-server / notification webhook

### ISSUE THREAD

**Issue by ayushin** [Tiledesk/tiledesk-server#43]
Hi,

We are evaluating moving our in-app chat to tiledesk - it is really nice that you support JWT (if you would also have a  widget/client in Flutter would be totally awesome, but anyway...)

The question is how to send a push notification to our app user to let them know they have a new message from support?

Is there a way to set up a webhook for this via https://console.tiledesk.com/v2/dashboard or API?

**Comment by andrealeo83** [comment]
yes use these webhook https://developer.tiledesk.com/apis/webhooks

**Comment by andrealeo83** [comment]
use message.create event to be notified for each message sent: https://developer.tiledesk.com/apis/webhooks#webhook-events
